pynxtools/testing/nexus_conversion.py

```diff
@@ -94,14 +95,17 @@ def convert_to_nexus(self, ignore_undocumented=False):
             template=Template(template), file_paths=tuple(input_files)
         )
 
+        entry_names = read_data.get_all_entry_names()  # type: ignore[attr-defined]
+        for entry_name in entry_names:
+            write_nexus_def_to_entry(read_data, entry_name, self.nxdl)
+
```

Comment:
This is not the part of the test. This is how you include the nexus definition version in the entry. That should be part of the general nexus converter so that every nexus file file gets the version by default.